app/src/main/java/org/oppia/android/app/profile/AdminSettingsDialogFragmentPresenter.kt

```diff
@@ -40,18 +41,19 @@ class AdminSettingsDialogFragmentPresenter @Inject constructor(
       viewModel = adminViewModel
     }
 
-    binding.inputPin.setInput(adminViewModel.inputPin.get().toString())
-    binding.inputPin.addTextChangedListener(object : TextWatcher {
-      override fun onTextChanged(confirmPin: CharSequence?, start: Int, before: Int, count: Int) {
-        confirmPin?.let {
-          adminViewModel.inputPin.set(confirmPin.toString())
+    var tempError = adminViewModel.errorMessage.get()
```

Comment:
Suggest not using `var`.

Also the naming is not that apt. suggest writing it as `adminInputPinError`

Reply:
@rt4914 here i'm using var as i'm changing it's value in `onTextWatched`
please check [this](https://github.com/oppia/oppia-android/pull/2349#discussion_r550956821) comment which explains my implementation

Reply:
Done